generateViewModels.js

```diff
@@ -0,0 +1,84 @@
+var https = require('https');
+var fs = require('fs');
+var path = require('path');
+var request = require('request');
+var jsYaml = require('js-yaml');
+var unzip = require('unzip2');
+
+var newsApiUrl = "https://dev.api.news.gov.bc.ca/swagger/v1/swagger.json";
+var hubApiUrl = "https://dev.api.hub.gov.bc.ca/swagger/v1/swagger.json";
```

Comment:
Is the hub api going to be public? Won't it need to be authed to build these models

Reply:
Yes, but we will already have the auth key in the Dashboard